pkg/operator/ceph/file/mds/mds_liveness_probe.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+ROOK_CEPH_MON_HOST="{{ .RookCephMonHost }}"
+ROOK_CEPH_MON_INITIAL_MEMBERS="{{ .RookCephMonInitialMembers }}"
+DAEMON_MDS="{{ .DaemonMds }}"
+FILESYSTEM_NAME="{{ .FilesystemName }}"
+
+outp="$(ceph fs dump --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --format json 2>&1)"
```

Comment:
Start collecting example output from this command for future unit testing. 

It'll be best if the example output is from 3 cases:
1. `ceph fs dump` returns no results (no fs'es)
2. `ceph fs dump` returns a single result (one fs)
3. `ceph fs dump` returns multiple results (multiple fs'es)

Reply:
It makes it a lot complicated, I don't think we need to test that all,

Added a standby test to it